XML/RegionAndStorageAccounts.xml

```diff
@@ -1,111 +1,111 @@
-        <PremiumStorage>eosgbrazilsouthx</PremiumStorage>
-    </brazilsouth>
-    <australiaeast>
-        <StandardStorage>eosgaustraliaeast</StandardStorage>
-        <PremiumStorage>eosgaustraliaeastx</PremiumStorage>
-    </australiaeast>
-    <australiasoutheast>
-        <StandardStorage>eosgaustraliasoutheast</StandardStorage>
-        <PremiumStorage>eosgaustraliasoutheastx</PremiumStorage>
-    </australiasoutheast>
-    <southindia>
-        <StandardStorage>eosgsouthindia</StandardStorage>
-        <PremiumStorage>eosgsouthindiax</PremiumStorage>
-    </southindia>
-    <centralindia>
-        <StandardStorage>eosgcentralindia</StandardStorage>
-        <PremiumStorage>eosgcentralindiax</PremiumStorage>
-    </centralindia>	
-    <westindia>
-        <StandardStorage>eosgwestindia</StandardStorage>
-        <PremiumStorage>eosgwestindiax</PremiumStorage>
-    </westindia>
-    <canadacentral>
-        <StandardStorage>eosgcanadacentral</StandardStorage>
-        <PremiumStorage>eosgcanadacentralx</PremiumStorage>
-    </canadacentral>
-    <canadaeast>
-        <StandardStorage>eosgcanadaeast</StandardStorage>
-        <PremiumStorage>eosgcanadaeastx</PremiumStorage>
-    </canadaeast>
-    <uksouth>
-        <StandardStorage>eosguksouth</StandardStorage>
-        <PremiumStorage>eosguksouthx</PremiumStorage>
-    </uksouth>
-    <ukwest>
-        <StandardStorage>eosgukwest</StandardStorage>
-        <PremiumStorage>eosgukwestx</PremiumStorage>
-    </ukwest>
-    <westcentralus>
-        <StandardStorage>eosgwestcentralus</StandardStorage>
-        <PremiumStorage>eosgwestcentralusx</PremiumStorage>
-    </westcentralus>
-    <westus2>
-        <StandardStorage>eosgwestus2</StandardStorage>
-        <PremiumStorage>eosgwestus2x</PremiumStorage>
-    </westus2>
-    <koreacentral>
-        <StandardStorage>eosgkoreacentral</StandardStorage>
-        <PremiumStorage>eosgkoreacentralx</PremiumStorage>
-    </koreacentral>
-    <koreasouth>
-        <StandardStorage>eosgkoreasouth</StandardStorage>
-        <PremiumStorage>eosgkoreasouthx</PremiumStorage>
-    </koreasouth>
-    <francecentral>
-        <StandardStorage>eosgfrancecentral</StandardStorage>
-        <PremiumStorage>eosgfrancecentralx</PremiumStorage>
-    </francecentral>
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<AllRegions>
+  <eastasia>
+    <StandardStorage>eosgeastasia</StandardStorage>
+    <PremiumStorage>eosgeastasiax</PremiumStorage>
+  </eastasia>
+  <southeastasia>
+    <StandardStorage>eosgsoutheastasia</StandardStorage>
+    <PremiumStorage>eosgsoutheastasiax</PremiumStorage>
+  </southeastasia>
+  <centralus>
+    <StandardStorage>eosgcentralus</StandardStorage>
+    <PremiumStorage>eosgcentralusx</PremiumStorage>
+  </centralus>
+  <eastus>
+    <StandardStorage>eosgeastus</StandardStorage>
+    <PremiumStorage>eosgeastusx</PremiumStorage>
+  </eastus>
+  <eastus2>
+    <StandardStorage>feliuuse2</StandardStorage>

```

Comment:
@Francis-Liu @johnsongeorge-w Include this file change by mistake?

Reply:
https://github.com/microsoft/lisa/pull/1366 reverts this changes.